abstract_specification.md

```diff
@@ -40,7 +40,7 @@ In this document, the term `dataframe` implies something akin to an Arrow `Table
 - multiple columns may exist, each with a string column name
 - all columns are individually typed and contain simple data types (e.g., int64)
 - all columns are of equal length
-- rows are addressed by offset (SOMADataFrame), or by one or more dataframe columns (SOMAIndexedDataFrame)
+- rows are addressed by one or more dataframe columns (SOMADataFrame)
```

Comment:
You can remove the parenthetical at the end of the sentence, as it is no longer required to disambiguate the clauses...